code/modules/examine/descriptions/weapons.dm

```diff
@@ -0,0 +1,81 @@
+/*
+	Note: This file is meant for actual weapons (guns, swords, etc), and not the stupid 'every obj is a weapon, except when it's not' thing.
+*/
+
+//******
+//*Guns*
+//******
+
+//This contains a lot of copypasta but I'm told it's better then a lot of New()s appending the var.
+/obj/item/weapon/gun
+	description_info = "This is a gun.  To fire the weapon, ensure your intent is *not* set to 'help', have your gun mode set to 'fire', \
```

Comment:
I'm not sure if this is a thing on paradise, I think guns still fire even when on help intent